packages/elements-core/src/components/Docs/Article/index.tsx

```diff
@@ -2,16 +2,22 @@ import { IMarkdownViewerProps } from '@stoplight/markdown-viewer';
 import { Box } from '@stoplight/mosaic';
 import { withErrorBoundary } from '@stoplight/react-error-boundary';
 import * as React from 'react';
+import { useLocation } from 'react-router-dom';
 
+import { RouterTypeContext } from '../../../context/RouterType';
 import { MarkdownViewer } from '../../MarkdownViewer';
 import { DocsComponentProps } from '..';
 
 type ArticleProps = DocsComponentProps<IMarkdownViewerProps['markdown']>;
 
 const ArticleComponent = React.memo<ArticleProps>(({ data }) => {
+  const { pathname } = useLocation();
+  const routerKind = React.useContext(RouterTypeContext);
+  const basePath = routerKind === 'hash' ? `#${pathname.split('#')[0]}` : '';
```

Comment:
```suggestion
  const basePath = routerKind === 'hash' ? `${hash}` : '';
```

Reply:
Also just noticed the `#` before the pathname. Was that a typo or am I missing somthing?

Reply:
no, that's expected.
We use double hashes for hash router.

<img width="604" alt="image" src="https://user-images.githubusercontent.com/9273484/223881045-aa97793b-4911-4186-82cf-8f223eb13381.png">

http://localhost:63342/elements/a.html#/cyrh2afdvwnpb-test#baz

the first part is the actual route, while the other hash is the anchor, hence the split.
We split to get the route and append the anchor.

Reply:
I see. Thanks for the clarification